src/cascadia/TerminalApp/TabPaletteItem.idl

```diff
@@ -12,10 +13,6 @@ namespace TerminalApp
 
         TabBase Tab { get; };
 
-        Boolean IsProgressRingActive { get; set; };
-        Boolean IsProgressRingIndeterminate { get; set; };
-        UInt32 ProgressValue { get; set; };
-        Boolean IsPaneZoomed { get; set; };
-        Boolean BellIndicator { get; set; };
+        TerminalTabStatus TabStatus { get; };

```

Comment:
Ah. I suppose I figured that `TabStatus` would be an interface, so you'd have something like

```
runtimeclass TabPaletteItem : PaletteItem, ITabStatus
```

which would force `TabPaletteItem` to implement all of the `IsProgressRingActive` etc declared in `ITabStatus`.

THAT BEING SAID,

This implementation does actually seem simpler to me. It's so darn convenient to just bind them both to one singular object.

Reply:
I started with the interface, but it still required the TabPaletteItem to register each property manually. So I switched to this.